rl4co/envs/routing/mpdp/env.py

```diff
@@ -14,51 +14,57 @@
 from rl4co.utils.ops import gather_by_index
 from rl4co.utils.pylogger import get_pylogger
 
+from .generator import MPDPGenerator
+from .render import render
+
 log = get_pylogger(__name__)
 
 
 class MPDPEnv(RL4COEnvBase):
-    """Multi-agent Pickup and Delivery Problem environment.
+    """Multi-agent Pickup and Delivery Problem (mPDP) environment.
```

Comment:
Note that actually this should be called "mPDTSP", but I guess we can leave it like that for now